src/App.js

```diff
@@ -1,12 +1,61 @@
-import React from "react";
+import React, { useState } from "react";
 import "./App.css";
 
-function App () {
+function App() {
+  const [score, setScore] = useState(0);
+  const [x, setX] = useState(1);
+  const [win, setWin] = useState(false);
```

Comment:
You should be able to tell if someone one by checking what their score is, so there should be no need to create a separate state value for this!